R/transforms.R

```diff
@@ -354,13 +354,13 @@ setMethod(">=", signature(e1= "numeric", e2= "IterableMatrix"), function(e1, e2)
 setClass("TransformRound", contains = "TransformedMatrix")
 
 setMethod("iterate_matrix", "TransformRound", function(x) {
-  iterate_matrix_round_cpp(iterate_matrix(x@matrix))
+  iterate_matrix_round_cpp(iterate_matrix(x@matrix), x@global_params[1])
 })
 
 setMethod("short_description", "TransformRound", function(x) {
   c(
     short_description(x@matrix),
-    "Transform round to nearest integer"
+    "Transform round to specified precision"
```

Comment:
Could we make this a bit more descriptive using `sprintf`?, like `sprintf("Transform round to %d decimal places", x@global_params[1])`